### DISCUSSION TITLE
Making an official OrcaSlicer discord?

### DISCUSSION THREAD

**Discussion by jeremytodd1** [OrcaSlicer/OrcaSlicer#1691] [General]
There is currently a discord server for OrcaSlicer, but it's not official. The creator of that discord has also moved away from OrcaSlicer and is now working on his own branch of the slicer.

I think it'd be a good idea to have an official Discord for the slicer.

**Comment by SoftFever** [comment]
Done!
![discord-mark-blue](https://github.com/SoftFever/OrcaSlicer/assets/103989404/b97d5ffc-072d-4d0a-bbda-e67ef373876f)  [OrcaSlicer Official Discord Server](https://discord.gg/WTEkCR6SnV)
